packages/wdio-cli/tests/utils.test.js

```diff
@@ -19,23 +18,8 @@ jest.mock('child_process', function () {
     }
     return m
 })
-global.console.log = jest.fn()
 
-test('getNpmPackageName', () => {
-    const reporters = [
-        ' dot - https://www.npmjs.com/package/@wdio/dot-reporter',
-        ' spec - https://www.npmjs.com/package/@wdio/spec-reporter',
-        ' junit - https://www.npmjs.com/package/@wdio/junit-reporter',
-        ' random - https://www.npmjs.com/package/wdio-random-reporter'
-    ]
-    expect(getNpmPackageName(reporters)).toEqual([
-        '@wdio/dot-reporter',
-        '@wdio/spec-reporter',
-        '@wdio/junit-reporter',
-        'wdio-random-reporter'
-    ])
-    expect(getNpmPackageName(' mocha - https://www.npmjs.com/package/wdio-mocha-framework')).toEqual('wdio-mocha-framework')
-})
+global.console.log = jest.fn()
```

Comment:
let's move this into beforeEach and restore the mock in afterEach otherwise if you use `console.log` in another test you will wonder why nothing is printed in the console